docs/development-getting-started/debugging-of-java-backend.md

```
@@ -13,85 +14,106 @@ Tips：
 
-![](./imgs/java-debugging-image34.png)
+如下图所示
 
-- 在项目根目录下执行启动`pnpm run serve:frontend`
+![](./imgs/java-debugging-image37.png)
 
-出现如下图所示表示启动成功，启动成功后浏览器会自动打开设计器页面
+#### 1.2 执行 SQL 文件导入数据库表与基础数据
 
-![](./imgs/java-debugging-image35.png)
+**在按照上述操作后的新建的数据库名称上右键选择“运行 SQL 文件”->在文件项选择下面两个 SQL 文件导入运行**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)